sankey.py

```diff
@@ -69,19 +69,32 @@ def sankey(left, right, leftWeight=None, rightWeight=None, colorDict=None,
     plt.rc('font', family='serif')
 
     # Create Dataframe
+    if isinstance(left, pd.Series):
+        left = left.reset_index(drop=True)
+    if isinstance(right, pd.Series):
+        right = right.reset_index(drop=True)
     df = pd.DataFrame({'left': left, 'right': right, 'leftWeight': leftWeight,
                        'rightWeight': rightWeight}, index=range(len(left)))
+    
+    if len(df[(df.left.isnull()) | (df.right.isnull())]):
+        raise RuntimeError('Sankey graph does not support null values.')
 
     # Identify all labels that appear 'left' or 'right'
     allLabels = pd.Series(np.r_[df.left.unique(), df.right.unique()]).unique()
 
     # Identify left labels
     if len(leftLabels) == 0:
         leftLabels = pd.Series(df.left.unique()).unique()
+    else:
+        if set(leftLabels) != set(df['left'].unique().tolist()):
+            raise RuntimeError('Left labels and left data do not match.')
```

Comment:
Would it be possible to add detail  why we have a run time ? Displaying the value for left labels and left data for example. Or number of elements if huge.

Reply:
Not opposed to displaying the labels and unique data. What constitutes "huge"? More than 10? 20?

Reply:
If we want to prevent to have more than a console screen of error message, 20 sound about right :)